src/Stack/Options/NewParser.hs

```diff
@@ -23,9 +23,12 @@ newOptsParser = (,) <$> newOpts <*> initOptsParser
              help "Do not create a subdirectory for the project") <*>
         optional (templateNameArgument
             (metavar "TEMPLATE_NAME" <>
-             help "Name of a template or a local template in a file or a URL.\
-                  \ For example: foo or foo.hsfiles or ~/foo or\
-                  \ https://example.com/foo.hsfiles")) <*>
+             help "Name of a template - can take the form\
+                \ [service:][username/]template with optional service name\
```

Comment:
Perhaps more accurate would be:

```
[[service:]username/]template
```

Reply:
No, actually you can write github:foo and it will pull the template `foo` from GitHub with the username `commercialhaskell` in the `stack-templates` repo. So either the service or the username can be dropped.

Reply:
Thanks for clarifying. In that case, I think it would be better to change the behavior, otherwise we'll be granting special status to the commercialhaskell account on other services when we don't necessarily control it.

Reply:
Yeah, I thought that might be odd too. I'll change the code.